### DISCUSSION TITLE
Unable to preview .json file with character '#' in path

### DISCUSSION THREAD

**Discussion by EarthMessenger** [spencerwooo/onedrive-vercel-index#412] [Q&A] [answered]
无法打开路径中含有 '#‘ 字符的 .json 文件。  
Unable to preview .json file with character '#' in path

报错如下 The error is as follows:

![图片](https://user-images.githubusercontent.com/49364506/153588956-c28e96d5-bde0-4ae4-a032-012b9b5ca2e6.png)

不过我非常有理由相信绝对不是因为 Too much requests。当我使用 microsoft graph api 来直接访问时仍然有同样的错误  
But I'm sure it's not because of too much requests. When I use microsoft graph api to access file, the same error still happened:

![图片](https://user-images.githubusercontent.com/49364506/153590874-b9a3577a-ba20-428c-bd21-8ea94832ba1d.png)

样例 example:
- https://onedrive.earthmessenger.xyz/zh-CN/test/%23.json
- https://onedrive.earthmessenger.xyz/zh-CN/test/%23/a.json
- https://onedrive.earthmessenger.xyz/zh-CN/test/~!%40%23%24%25%5E%26()_%2B%7B%7D%60-%3D%5B%5D%3B'%2C/a.json
- https://onedrive.earthmessenger.xyz/zh-CN/test/abc%23efg/a.json

这十分的奇怪，仿佛只发生在 .json 文件上，是因为转义的原因吗？  
That's quite strange. It seems only happen on .json files. Is it because of escaping of character '#'?

**Comment by myl7** [comment, marked as answer]
How about this offical support? <https://support.microsoft.com/en-us/office/restrictions-and-limitations-in-onedrive-and-sharepoint-64883a5d-228e-48f5-b3d2-eb39e07630fa#:~:text=Some%20organizations%20don%27t%20yet%20support%20%23%20and%20%25%20in%20names.%20If%20you%27re%20a%20global%20admin%20or%20SharePoint%20admin%20in%20Microsoft%20365%2C%20see%20Enabling%20%23%20and%20%25%20Support%20on%20the%20SharePoint%20blog%20to%20learn%20how%20to%20allow%20these%20characters.>
